From: Soc Virnyl Estela <socvirnyl.estela@gmail.com>
Date: Fri, 04 Aug 2023 14:59:44 PST
We invoke LLD so we also commented/remove it out as well.
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 92be8fb..e19d18e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,7 +126,7 @@ set(ZIG_PIE off CACHE BOOL "produce a position independent zig executable")
 
 find_package(llvm 16)
 find_package(clang 16)
-find_package(lld 16)
+# find_package(lld 16)
 
 if(ZIG_STATIC_ZLIB)
     if (MSVC)
@@ -173,7 +173,7 @@ foreach(CONFIG_TYPE ${CMAKE_CONFIGURATION_TYPES})
 endforeach(CONFIG_TYPE CMAKE_CONFIGURATION_TYPES)
 
 include_directories(${LLVM_INCLUDE_DIRS})
-include_directories(${LLD_INCLUDE_DIRS})
+# include_directories(${LLD_INCLUDE_DIRS})
 include_directories(${CLANG_INCLUDE_DIRS})
 
 find_package(Threads)
@@ -684,7 +684,7 @@ set_target_properties(zigcpp PROPERTIES COMPILE_FLAGS ${ZIGCPP_CXX_FLAGS})
 
 target_link_libraries(zigcpp LINK_PUBLIC
     ${CLANG_LIBRARIES}
-    ${LLD_LIBRARIES}
+    # ${LLD_LIBRARIES}
     ${LLVM_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT}
 )
